### ISSUE TITLE
Allow comparing results

### ISSUE THREAD

**Issue by LRuffati** [sixtriangles/sixtriangles.github.io#9]
It'd be nice to be able to compare results on the same picture, this could be allowed by concatenating different tokens in the url, something like:
`https://sixtriangles.github.io/results.html?token1_token2_...`

and having a form in the result page asking for a token to compare with and adding it to the URL

Once the token is read all that would be needed is drawing new circles in a different color

**Comment by SearchingForAPast** [comment]
I agree, this would be a nice feature, but I don't have the time to add it in the near future, I might do that in a few months, or if I find some free time.

**Comment by sixtriangles** [comment]
I agree, this would be a nice feature, but I don't have the time to add it in the near future, I might do that in a few months, or if I find some free time
